### DISCUSSION TITLE
Welcome to spix Discussions!

### DISCUSSION THREAD

**Discussion by faaxm** [faaxm/spix#26] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by hstig** [comment]
How can I enter text into a cell of table view?
